extensions/amp-ad-network-smartads-impl/0.1/smartads-a4a-config.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {startsWith} from '../../../src/string';
+
+/** @const @private {string} */
+const SRC_PREFIX_ = 'https://smart-ads.biz/_amp';
+
+/** @const @private {string} */
+const SRC_A4A_PREFIX_ = 'https://smart-ads.biz/_a4a';
+
+/**
+ * @param {!Window} win
+ * @param {!Element} element
+ * @param {boolean} useRemoteHtml
+ * @return {boolean}
+ */
+export function smartAdsIsA4AEnabled(win, element, useRemoteHtml) {
+  let src;
+  return (
+    !useRemoteHtml &&
+    !!(src = element.getAttribute('src')) &&
```

Comment:
Is `=` intended?

Reply:
I don 't quite understand

Reply:
Can this assignment be moved to below (line 32)? The logic here is a bit difficult to understand.

```
let src = element.getAttribute('src');
return (
    !useRemoteHtml &&
    !!src &&
    ....
```

Reply:
good idea